packages/api/core/src/util/forge-config.ts

```diff
@@ -124,8 +122,11 @@ export default async (dir: string): Promise<ResolvedForgeConfig> => {
   if (await forgeConfigIsValidFilePath(dir, forgeConfig)) {
     try {
       // eslint-disable-next-line @typescript-eslint/no-var-requires
-      const loaded = require(path.resolve(dir, forgeConfig as string)) as MaybeESM<ForgeConfig>;
-      forgeConfig = 'default' in loaded ? loaded.default : loaded;
+      let loaded = require(path.resolve(dir, forgeConfig as string));
```

Comment:
question: How come we don't need the `MaybeESM` type anymore?

Reply:
So `loaded` can now be three different types - a `ForgeConfig`, `MaybeESM`, or a `Function`. It was starting to get a bit into TypeScript hell to account for all three types, but I can take a stab at making that typing more precise, just so we're clear on what we're potentially handling 👀

Reply:
Added more detailed typing, I like this much more 😄 Open to any suggestions from TypeScript savvy folks